gitter/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,4 @@
+$ErrorActionPreference = 'Stop'; # stop on all errors
+
+# Remove the Start Menu shortcuts from the All Users profile we created at install
+Remove-Item -Path (Join-Path -Path $env:ALLUSERSPROFILE -ChildPath 'Microsoft\Windows\Start menu\Programs\Gitter') -Recurse
```

Comment:
Just installed gitter on another computer and noticed my PR hasn't been merged.

Just in case my previous comment was missed Gitter is uninstalled automatically by Choco and the chocolateyUninstall.ps1 is then called to remove the shortcuts etc. So you don't have to explicitly uninstall it.